src/vs/workbench/api/node/extHost.api.impl.ts

```diff
@@ -660,6 +660,9 @@ export function createApiFactory(
 			onDidCloseTextDocument: (listener, thisArgs?, disposables?) => {
```

Comment:
This event was named onDidCloseTextDocument, but it's linked to an event named onDidRemoveDocument that is actually the right name for it because it's fired when the document is removed, not closed. Don't know what would be the right way to rename it, as it's an API used by many extensions.